lib/bundler_api/gem_info.rb

```diff
@@ -2,7 +2,7 @@
 
 # Return data about all the gems: all gem names, all versions of all gems, all dependencies for all versions of a gem
 class BundlerApi::GemInfo
-  DepKey = Struct.new(:name, :number, :platform)
+  DepKey = Struct.new(:name, :number, :platform, :required_ruby_version, :rubygems_version)
```

Comment:
Is this the correct place for those? I've ended up changing a lot of stuff because of those.